src/modifiers/selection_type.py

```diff
@@ -25,13 +25,13 @@ def json_repr(self):
 
 TOKEN = SelectionType("token", "tokens", "token", 0)
 LINE = SelectionType("line", "lines", "line", 1)
-BLOCK = SelectionType("block", "blocks", "block", 2)
+BLOCK = SelectionType("block", "blocks", "paragraph", 2)
```

Comment:
i think we need a rebase?

Reply:
why? this change is already done extension side.

Reply:
Because I'm pretty sure this change is already on master and develop, no?

Reply:
You're completely right have no idea how that had gotten in there in two different branches. Fixed now.